app/utils/dictionary/payment.js

```diff
@@ -6,174 +6,199 @@ export const paymentCurrencies = [
     code   : 'PLN',
     symbol : 'zł',
     name   : 'Polish zloty',
-    stripe : true
+    stripe : true,
+    omise  : true
```

Comment:
These are wrong. Please see omise's documentation. They have a very limited support in terms of currencies. Kindly use `omise: true` in the applicable positions only.